Mathlib/MeasureTheory/Group/GeometryOfNumbers.lean

```diff
@@ -58,13 +58,15 @@ theorem exists_pair_mem_lattice_not_disjoint_vadd [AddCommGroup L] [Countable L]
       (measure_mono <| Set.iUnion_subset fun _ => Set.inter_subset_right _ _)
 #align measure_theory.exists_pair_mem_lattice_not_disjoint_vadd MeasureTheory.exists_pair_mem_lattice_not_disjoint_vadd
 
+variable [NormedAddCommGroup E] [NormedSpace ℝ E] [BorelSpace E] [FiniteDimensional ℝ E]
+  [IsAddHaarMeasure μ] {L : AddSubgroup E} [Countable L] (fund : IsAddFundamentalDomain L F μ)
+   (h_symm : ∀ x ∈ s, -x ∈ s) (h_conv : Convex ℝ s)
+
 /-- The **Minkowski Convex Body Theorem**. If `s` is a convex symmetric domain of `E` whose volume
 is large enough compared to the covolume of a lattice `L` of `E`, then it contains a non-zero
 lattice point of `L`.  -/
-theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure [NormedAddCommGroup E]
-    [NormedSpace ℝ E] [BorelSpace E] [FiniteDimensional ℝ E] [IsAddHaarMeasure μ]
-    {L : AddSubgroup E} [Countable L] (fund : IsAddFundamentalDomain L F μ)
-    (h : μ F * 2 ^ finrank ℝ E < μ s) (h_symm : ∀ x ∈ s, -x ∈ s) (h_conv : Convex ℝ s) :
+theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure
+    (h : μ F * 2 ^ finrank ℝ E < μ s) :
```

Comment:
could you move the Prop variables back to the statement? I know they are common to this theorem and the next one, so it is tempting to factor them out, but in practice it makes the statements much harder to read. A good rule of thumb is that explicit Prop variables should be in the body of the theorem 99% percent of the time.

Reply:
I can see your point especially with so many hypothesis. Done.